.test.mk

```diff
@@ -129,7 +129,21 @@ test-coverage: build run-luajit-test run-test
 	     --exclude '/usr/*' \
 	     --exclude '*/build/*' \
 	     --exclude '*/test/*' \
-	     --exclude '*/third_party/*'
+	     --exclude '*/third_party/c-ares' \
```

Comment:
It would be nice to leave a comment why we exclude exactly these third-party projects but not all.

Reply:
Yes, I saw your description in the commit message, but it would be nice to have it here as well. In a half year, when I look at this part of the code, I would like to see why it is done without searching the commit.